modules/core/src/ocl.cpp

```diff
@@ -6370,7 +6370,6 @@ class OpenCLAllocator CV_FINAL : public MatAllocator
 static OpenCLAllocator* getOpenCLAllocator_() // call once guarantee
 {
     static OpenCLAllocator* g_allocator = new OpenCLAllocator(); // avoid destructor call (using of this object is too wide)
-    g_isOpenCVActivated = true;
```

Comment:
Not necessary too.

`UMat::getStdAllocator()` triggers this flag through `ocl::useOpenCL()` call.